### DISCUSSION TITLE
Support for attribute filters

### DISCUSSION THREAD

**Discussion by kapso** [okuramasafumi/alba#330] [Ideas]
Any plans for supporting filters - https://panko.dev/docs/attributes#filters

So something like 

```ruby
BlogResource.new(
  blog, 
  only: {
    blog: [:id, :content],
    comment: [:id, :text, :by_user]
  }
).serialize
```

**Comment by okuramasafumi** [comment]
Basically you want to the same as https://github.com/okuramasafumi/alba/discussions/326
Please refer to that answer.
